kubevirt/schema/virtualmachine/spec.go

```diff
@@ -3,18 +3,17 @@ package virtualmachine
 import (
 	"fmt"
 
-	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
-	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
-	"github.com/kubevirt/terraform-provider-kubevirt/kubevirt/schema/datavolume"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
 	"github.com/kubevirt/terraform-provider-kubevirt/kubevirt/schema/virtualmachineinstance"
-	kubevirtapiv1 "kubevirt.io/client-go/api/v1"
+	kubevirtapiv1 "kubevirt.io/api/core/v1"
 )
 
 func virtualMachineSpecFields() map[string]*schema.Schema {
 	return map[string]*schema.Schema{
 		// "running": &schema.Schema{
```

Comment:
Why does the attribute `running` in a comment? can you remove it?

Reply:
There was an issue adding the "running" with "run_strategy"
Should be fixed in this issue https://github.com/kubevirt/terraform-provider-kubevirt/issues/15